Bottom: 6e8ecc04d243cce9376c70dc2c42dceca40fee22
Top:    9d619ed5215c6596d6cdb69e00586695e39930f5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-30 00:24:02 +0100

Refresh of draw-c-expose-some-drawing-api

---

diff --git a/src/draw.h b/src/draw.h
index c0a2bb1..d92dee5 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -76,5 +76,6 @@ void DrawPaste (int side, const BoxType *drawn_area);
 void DrawSilk (int side, const BoxType *drawn_area);
 void DrawMask (int side, const BoxType *drawn_area);
 void DrawHoles (bool draw_plated, bool draw_unplated, const BoxType *drawn_area);
+void PrintAssembly (int side, const BoxType *drawn_area);
 
 #endif
